static/styles/zulip.css

```diff
@@ -124,6 +124,7 @@ pre {
 
     .tippy-content {
         font-size: 13px;
+        font-weight: 400;
```

Comment:
Can you explain this change? It seems like it'd apply to _all_ tooltips in the app.

Reply:
Without it the tooltip content had the bold effect, 
I have fixed it by moving it to `recent_topics.css`.